edx_proctoring/templates/proctoring/seq_proctored_exam_verified.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+<div class="success sequence proctored-exam passed" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Your proctoring session was reviewed and passed all requirements
+    {% endblocktrans %}
+  </h3>
+
+  <h4>
+    {% blocktrans %}
+      Your Proctoring Session review: <b class="success"> Passed </b>
+    {% endblocktrans %}
+  </h4>
+
+  <p>
+    {% blocktrans %}
+      This satisfies the requirement that you complete this exam in a proctored format and contributes
```

Comment:
"You have completed a proctored exam with a **Satisfactory** proctoring session result. You must also achieve a passing score in the exam to be eligible to use this course for academic credit."
@griffresch OK?